civictechprojects/views.py

```diff
@@ -581,6 +581,40 @@ def recent_projects_list(request):
         return JsonResponse({'projects': hydrated_project_list})
 
 
+def limited_listings(request):
+    """Summarizes current positions in a format specified by the LinkedIn "Limited Listings" feature."""
+
+    def cdata(str):
+        # Using CDATA tags (and escaping the close sequence) protects us from XSS attacks when
+        # displaying user provided string values.
+        return f"<![CDATA[{str.replace(']]>', ']]]]><![CDATA[>')}]]>"
+    
+    def position_to_job(position):
+        project = position.position_project
+        roleTag = Tag.get_by_name(position.position_role.first().slug)
+
+        return f"""
+        <job>
+            <company>{cdata(project.project_name)}</company>
+            <title>{cdata(roleTag.display_name)}</title>
+            <description>{cdata(position.position_description)}</description>
+            <location>{cdata(project.project_location)}</location>
```

Comment:
This location field is a code specific to HERE.  We'll want to use the city, state, and country values instead.

Reply:
In the spec, it's described: `This is a description of the job's location (e.g. “San Francisco, CA”). LinkedIn will automatically map this to a country and postal code unless those fields are provided. ` Do you mean the value we have stored is different from the values in `testdata.json`?

If we want to populate city/state/country, is it safe to parse the `project.project_location` field and assume it's in the US? Or did you have another source for those fields in mind?

Reply:
The testdata.json info currently in the master branch is out of date when it comes to location (see https://github.com/DemocracyLab/CivicTechExchange/pull/407 for updated test data).  To populate this now, you can use project_country, project_state, project_city.

Reply:
Ahh, I see. I've merged from master, makes a lot more sense now.